Invalid calculation in `test_not_announceable_spend`

```python
        # Hey, this spend is huge!
        with pytest.raises(
            RpcError, match="Spend transaction is too large, try spending less outpoints'"
        ):
            man.rpc.getspendtx(deposits, destinations, feerate)
    
        # One less output is ok though
        n_outputs -= 1
        destinations = {
            bitcoind.rpc.getnewaddress(): (total_amount - fees) // n_outputs
            for _ in range(n_outputs)
        }
        spend_tx = man.rpc.getspendtx(deposits, destinations, feerate)["spend_tx"]
        for man in revault_network.mans():
            spend_tx = man.man_keychain.sign_spend_psbt(spend_tx, deriv_indexes)
            man.rpc.updatespendtx(spend_tx)
        spend_psbt = serializations.PSBT()
        spend_psbt.deserialize(spend_tx)
        spend_psbt.tx.calc_sha256()
>       man.rpc.setspendtx(spend_psbt.tx.hash)

tests/test_misc.py:2145: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests/test_framework/utils.py:352: in wrapper
    return self.call(name, payload=args)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <test_framework.utils.UnixDomainSocketRpc object at 0x7fb3cc4e6e80>, method = 'setspendtx'
payload = ('54e3d0a31795eef807675dd20d6a26f912e239fae00f2b4c160e26dd56d930a5',)

    def call(self, method, payload=[]):
        self.logger.debug("Calling %s with payload %r", method, payload)
    
        # FIXME: we open a new socket for every readobj call...
        sock = UnixSocket(self.socket_path)
        msg = json.dumps(
            {
                "jsonrpc": "2.0",
                "id": 0,
                "method": method,
                "params": payload,
            }
        )
        sock.sock.send(msg.encode())
        this_id = self.next_id
        resp = self._readobj(sock)
    
        self.logger.debug("Received response for %s call: %r", method, resp)
        if "id" in resp and resp["id"] != this_id:
            raise ValueError(
                "Malformed response, id is not {}: {}.".format(this_id, resp)
            )
        sock.close()
    
        if not isinstance(resp, dict):
            raise ValueError(
                "Malformed response, response is not a dictionary %s." % resp
            )
        elif "error" in resp:
>           raise RpcError(method, payload, resp["error"])
E           test_framework.utils.RpcError: RPC call failed: method: setspendtx, payload: ('54e3d0a31795eef807675dd20d6a26f912e239fae00f2b4c160e26dd56d930a5',), error: {'code': -32602, 'message': "Communication error while announcing the Spend transaction: Network error: 'Noise Error: Message too large to encrypt: '97372' bytes but max is '65501''"}
```

We need to get rid of the randomness..

--- a/tests/test_spend.py
+++ b/tests/test_spend.py
@@ -565,7 +565,7 @@ def test_large_spends(revault_network, bitcoind, executor):
 # (it wouldn't be possible to announce it to the coordinator when fully signed)
 @pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
 def test_not_announceable_spend(revault_network, bitcoind, executor):
-    CSV = 2016  # 2 weeks :tm:
+    CSV = 2
     revault_network.deploy(17, 16, csv=CSV)
     man = revault_network.man(0)
 
@@ -622,11 +622,7 @@ def test_not_announceable_spend(revault_network, bitcoind, executor):
     )
 
     # We'll broadcast the Spend transaction as soon as it's valid
-    # Note that bitcoind's RPC socket may timeout if it needs to generate too many
-    # blocks at once. So, spread them a bit.
-    for _ in range(10):
-        bitcoind.generate_block(CSV // 10)
-    bitcoind.generate_block(CSV % 10 - 1)
+    bitcoind.generate_block(CSV)
     man.wait_for_log(
         f"Succesfully broadcasted Spend tx '{spend_psbt.tx.hash}'",
     )

